messaging.rst

```diff
@@ -4,7 +4,36 @@ Raiden Messages Specification
 Overview
 ========
 
-This is the specification document for the messages used in the Raiden protocol.
+This is the specification of the messages used in the Raiden protocol.
+
+There are data structures which reappear in different messages:
+The :ref:`offchain balance proof <balance-proof-offchain>`
```

Comment:
can you have those 2 in a numbered list?

Reply:
I did a bullet-point list to be consistent with the other overview sections.

Reply:
ok, sure